app/components/networking/address-space.tf

```diff
@@ -1,15 +1,11 @@
-module "common_vnet_address_space" {
+module "vnet_address_space" {
   source          = "hashicorp/subnets/cidr"
   version         = "1.0.0"
-  base_cidr_block = var.common_vnet_address_space
+  base_cidr_block = var.base_cidr_block
   networks = [
-    {
-      name     = "app_gateway"
-      new_bits = 9 # /25 (123 usable) [0 - 127]

```

Comment:
Interested to see if this will still deploy. If not, we can optionally change the `vnet_gateway` subnet to a /24 (`new_bits = 8`), or add the `app_gateway` /25 back in but enter the name as `null` to effectively reserve the space.

Reply:
Looks like it works!